src/main/java/com/semmle/jira/addon/util/JiraUtils.java

```diff
@@ -52,6 +52,17 @@ public static void addWorkflowToProject(Project project, String workflowName, Is
     workflowSchemeManager.addWorkflowToScheme(workflowScheme, workflowName, issueType.getId());
   }
 
+  public static IssueType getLgtmIssueType() {
+    Collection<IssueType> allIssueTypes =
+        ComponentAccessor.getConstantsManager().getAllIssueTypeObjects();
+    for (IssueType issueType : allIssueTypes) {
```

Comment:
Might be able to use `ComponentAccessor.getConstantsManager().getIssueConstantByName(ConstantsManager.CONSTANT_TYPE.ISSUE_TYPE, Constants.ISSUE_TYPE_NAME)`